atd-etl/app/process_cris_request.py

```diff
@@ -76,30 +76,45 @@ def wait(int):
     "Selecting data extract request, from '%s' to '%s'"
     % (CRIS_EXTRACT_DATE_START, CRIS_EXTRACT_DATE_END)
 )
-browser.find_by_text("Create Data Extract Request").click()
+start_button = browser.find_by_text("Create Data Extract Request")
+start_button.click()
 wait(10)
 
-browser.find_by_text("Continue").click()
+browser.find_by_text("Next").click()
 wait(10)
 
 print("Selecting Counties to be Included in the Extract")
-browser.find_by_css('input[ng-value="shareConstants.LOCATION_TYPE_IDS.COUNTY"]').click()
-browser.execute_script("$(\"div[data-value='105']\").click()")  # Travis
-browser.execute_script("$(\"div[data-value='227']\").click()")  # Williamson
-browser.execute_script("$(\"div[data-value='246']\").click()")  # Hays
+browser.find_by_id('rdoCounties').click()
+# Open ng-select dropdown
+browser.find_by_id('requestCounties').click() 
+
+browser.find_by_text('TRAVIS').click()
+
+# Reopen dropdown
+browser.find_by_id('requestCounties').click() 
+browser.find_by_text('WILLIAMSON').click()
+
+# Reopen dropdown
+browser.find_by_id('requestCounties').click() 
+browser.find_by_text('HAYS').click()
```

Comment:
Glad you made it work in a more simple way :) far easier to read than injecting javascript and executing 🙏